src/mscorlib/mscorlib.csproj

```diff
@@ -181,7 +181,8 @@
   </PropertyGroup>
 
   <Import Project="$(MSBuildThisFileDirectory)Tools\Versioning\GenerateVersionInfo.targets"/>
-  <Import Project="GenerateSplitStringResources.targets"/>
+  <Import Project="GenerateSplitStringResources.Windows.targets" Condition="'$(OsEnvironment)'=='Windows_NT'"/>
```

Comment:
Yes.  When I do the work in TFS I will merge these files so there will be a single copy for open (always uses Python) and a different file providing the targets (in a closed source directory) to use ResGen when doing a TFS build.  Sound good?